.github/workflows/pytest.yml

```diff
@@ -22,18 +22,21 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.8", "3.8"]
```

Comment:
I think this needs to be 3.8, 3.9 and 3.10?

Reply:
yeah just pushed a change

Reply:
perfect, approved